src/foundry/foundry.js/pixi/filters/abstractFilter.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * An abstract filter which provides a framework for reusable definition
+ */
+declare abstract class AbstractFilter extends PIXI.Filter {
+  /**
+   * The default uniforms used by the filter
+   * @defaultValue `{}`
+   */
+  static defaultUniforms: AbstractBaseShader.Uniforms;
+
+  /**
+   * The fragment shader which renders this filter.
+   * @defaultValue `undefined`
+   */
+  static fragmentShader: string | undefined;
+
+  /**
+   * The vertex shader which renders this filter.
+   * @defaultValue `undefined`
+   */
+  static vertexShader: string | undefined;
+
+  /**
+   * A factory method for creating the filter using its defined default values
+   * @param uniforms - (default: `{}`)
+   */
+  static create(uniforms: AbstractBaseShader.Uniforms): AbstractFilter;
```

Comment:
Shouldn't the return type be `InstanceType<this>`? Or do we explicitly not care about the concrete Filter type?

Reply:
I only realized again when I just checked it in the code: This is static. There is no `this` available.